### PR TITLE
added branch, glek

### PR DESCRIPTION
**Pull request by Kafui123**
implemented functions and called them in the main function to draw a house.

### CODE REVIEW

File: hw03_glek.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        kkg.forward(45)
+        kkg.left(90)
+
+
+
+
+
+
+    pass
+    # ....
```
Comment (sheggen) [inline hw03_glek.py:66]: Remove help text and extra blank lines. Should be exactly 2 blank lines before the next function def.

File: hw03_glek.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    """
+    Omo = turtle.Turtle()
+    Omo.pensize(15)
+    Omo.penup()
+    Omo.goto(-250, -250)
+    Omo.pendown()
+    Omo.hideturtle()
+
+
+    pass
```
Comment (sheggen) [inline hw03_glek.py:81]: Remove these, they were placeholders so the functions didn't return a syntax error.

File: hw03_glek.py (hunk 1 above)
Comment (Kafui123) [reply hw03_glek.py]: Please I have removed the extra blank lines of code.

File: hw03_glek.py (hunk 2 above)
Comment (Kafui123) [reply hw03_glek.py]: Dear Sir,
Please I have removed the placeholders.
Thank you.

File: hw03_glek.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
+#################################################################################
+# Author: Kafui Gle
+# Username: glek
+#
+# Assignment: hw03
+# Purpose:
+# Google Doc Link: https://docs.google.com/document/d/1yGLLdFBhjPw96O7yJiaUsUuLIPZWXuOvJ6QDpzMWM_c/edit
```
Comment (sheggen) [inline hw03_glek.py:7]: Use the header block formatting provided to you in the template file.

File: hw03_glek.py [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    """
+    wn = turtle.Screen()
+    wn.colormode(255)
+    wn.bgcolor(145, 180, 167)   # this sets the background color of the screen using rgb values.
+    # Function calls to function_1 and function_2.
+    drawOmo()
+    drawKkg()
+    wn.exitonclick()
+
+    main()        # Starts the program!
```
Comment (sheggen) [inline hw03_glek.py:90]: This should be moved left one indentation, or else it never runs. How did you test your code? It never calls main!

File: hw03_glek.py (hunk 3 above)
Comment (Kafui123) [reply hw03_glek.py:7]: changed the header block formatting style to the one in the template file as suggested.

File: hw03_glek.py (hunk 4 above)
Comment (Kafui123) [reply hw03_glek.py:90]: Initially, when I tested the code out, it was still functioning normally. It has been moved left one indentation as required.

**Pull request merged**